projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/parser/splitter/ProjectViewRawSections.java

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.bsp.bazel.projectview.parser.splitter;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.apache.commons.collections4.CollectionUtils;
+
+public class ProjectViewRawSections {
+
+  private final List<ProjectViewRawSection> sections;
```

Comment:
```suggestion
  private final Stream<ProjectViewRawSection> sections;
```
For extra laziness. Though I'm not sure if streams as class members is a good pattern. But I see the code is using localised streams in quite a few places, so perhaps there are some opportunities to pass these around instead of lists.

Reply:
Ah of course...